release-team/role-handbooks/release-notes/editing-flow.md

```diff
@@ -0,0 +1,245 @@
+
+This document details how to use krel to review and edit the 
+Kubernetes release notes during the cycle. 
+
+## Motivation
+
+The final Kubernetes Release Notes document is a large document composed of
+hundreds of entries that the contributors write into their Pull Requests. As
+an example, the [release notes document for Kubernetes 1.19](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.19.md)
+has 388 entries.
+
+Before v1.20, the Release Notes Team had to wait until code freeze to review
+and edit each Kubernetes release note in a short time. During the same
+timespan, the Team was expected to contact the sigs and edit the Major themes.
+Work was unbalanced with little to do most of the cycle and a packed 
+agenda during the last few weeks. 
+
+To balance the team's efforts evenly, tooling was developed to enable a new workflow.
+
+## Workflow Operation
+
+Reviewing the release notes involves running krel regularly to review new release
+notes in incoming pull requests, possibly editing some of them, and submitting
+changes back to `sig-release`.
+
+### `krel release-notes --fix`
+
+The main interactive editing flow is started with the --fix flag. The interactive
+mode will show the user all release notes that need to be reviewed. After
+reviewing each note, the user can edit the note's attributes using their editor of
+choice.
+
+The `--fix` flag is available when invoking krel with the `--create-draft-pr` option.
+This is an example invocation to edit the release notes for Kubernetes v1.19:
+
+```bash
+krel release-notes --create-draft-pr --org=MyGitHubOrg --fix --tag=v1.19.0
+```
+As with all `krel release-notes` subcommands, a GitHub token must be exported in
+an environment variable called `GITHUB_TOKEN`. The user should supply their GitHub
+organization where krel will push changes and from where it will create pull
+requests.
+
+After invoking the subcommand, krel will clone kubernetes/kubernetes and query GitHub
+for all pull requests. Note that this process might take a long time, particularly
+at the end of the cycle when the branch has lots of commits.
+
+
+#### Starting the interactive mode
+
+When `--fix` is defined, krel will switch to interactive mode.
+It will print a short intro and prompt the user before starting:
+
+```
+Welcome to the Kubernetes Release Notes editing tool!
+
+This tool will allow you to review and edit all the release
+notes submitted by the Kubernetes contributors before publishing
+the updated draft.
+
+The flow will show each of the release notes that need to be
+reviewed once and you can choose to edit it or not.
+
+After you choose, it will be marked as reviewed and will not
+be shown during the next sessions unless you choose to do a
+full review of all notes.
+
+You can hit Ctrl+C at any time to exit the review process
+and submit the draft PR with the revisions made so far.
+
+Would you like to continue from the last session? (y/n):
+``` 
```

Comment:
If we change this text in krel, are we going to keep this text up to date in this document? Is it worth just saying that the tool will print some explanatory text? If it's wroth keeping it, maybe put a comment in krel's source code where this string is defined to remind future developers to update this documentation too?

Reply:
OK, noted. I'll add a comment in the next PR.